_layouts/challenge.html

```diff
@@ -12,6 +12,7 @@
                         <span class="icon-bar"></span>
                         <span class="icon-bar"></span>
                     </a>
+                    <a class="brand" style="color: white;" href="../">Punya</a>
```

Comment:
I would recommend making the href `/` here so that if we ever move the file we don't have to update the relative path.

Reply:
I agree with you.